src/main/scala-3/uzhttp/header/CompatMap.scala

```diff
@@ -0,0 +1,3 @@
+package uzhttp.header
+
+trait CompatMap[K, V] extends scala.collection.immutable.Map[K, V]
```

Comment:
Is this used anywhere?

Reply:
[Headers](https://github.com/polynote/uzhttp/blob/master/src/main/scala/uzhttp/header/Headers.scala) class extends CompatMap

Reply:
Oh, yes of course 🤦